orttraining/orttraining/training_ops/cuda/communication/nccl_service.cu

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if defined(USE_NCCL) || defined(USE_HOROVOD)
```

Comment:
>|| defined(USE_HOROVOD) [](start = 22, length = 23)

why?

Reply:
Removed.

---
In reply to: [483800074](https://github.com/microsoft/onnxruntime/pull/5054#discussion_r483800074) [](ancestors = 483800074)